src/db/schema/webhooks.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  integer,
+  sqliteTable,
+  text,
+  uniqueIndex,
+} from "drizzle-orm/sqlite-core";
+
+export const webhookTbl = sqliteTable(
+  "webhooks",
+  {
+    id: integer("id", { mode: "number" }).primaryKey({ autoIncrement: true }),
+    url: text("url").notNull(), // The URL where the webhook will be sent
+    secret: text("secret"), // Optional secret for signing webhook payloads
+    eventType: text("event_type").notNull(), // Event type the subscriber is interested in
```

Comment:
eventuelt lav lige `enum` også smid et par typer, også export typerne. du kan finde eksempel i `season.ts` `RatingSystem`

Reply:
tak!

Reply:
he added the thing wow